persistence/sql/persister_oauth2.go

```diff
@@ -281,8 +284,9 @@ func (p *Persister) deactivateSessionByRequestID(ctx context.Context, id string,
 	return sqlcon.HandleError(
 		p.Connection(ctx).
 			RawQuery(
-				fmt.Sprintf("UPDATE %s SET active=false WHERE request_id=?", OAuth2RequestSQL{Table: table}.TableName()),
+				fmt.Sprintf("UPDATE %s SET active=false WHERE request_id=? AND nid = ?", OAuth2RequestSQL{Table: table}.TableName()),
```

Comment:
QueryWith

Reply:
I'm not sure if it's possible to combine RawQuery and QueryWith. Would you elaborate on this suggestion?

Reply:
You're right, I don't think that's possible